test/Mono.Linker.Tests.Cases/RequiresCapability/RequiresWithLambdas.cs

```diff
@@ -0,0 +1,101 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using System.Security.Permissions;
+using System.Text;
+using System.Threading.Tasks;
+using Mono.Linker.Tests.Cases.Expectations.Assertions;
+using Mono.Linker.Tests.Cases.Expectations.Helpers;
+
+namespace Mono.Linker.Tests.Cases.RequiresCapability
+{
+	[SkipKeptItemsValidation]
+	public class RequiresWithLambdas
```

Comment:
Ideally all tests should have `ExpectNoWarnings` attribute.
In which case you may want to add a suppression onto the Main as it will produce lot of 2026 warnings - but that's OK and the suppression is also OK - this test doesn't verify the exact warnings for that scope.